app/components/channel_info/index.js

```diff
@@ -0,0 +1,420 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {injectIntl, intlShape} from 'react-intl';
```

Comment:
we are trying to get away from the `injectIntl` and use it in the context, the way to do it is

```js
static contextTypes = {
    intl: intlShape
};
```

and then you can just do `export default class ... `